x-pack/plugins/apm/server/routes/transactions.ts

```diff
@@ -152,10 +147,61 @@ export const transactionChartsRoute = createRoute({
       transactionName,
       setup,
       searchAggregatedTransactions,
-      logger,
     };
 
-    return getTransactionCharts(options);
+    const { latency, overallAvgDuration } = await getLatencyCharts(options);
+
+    const anomalyTimeseries = latency
+      ? await getAnomalySeries({
+          ...options,
+          logger,
+          timeSeriesDates: latency.avg.map(({ x }) => x),
+        })
+      : null;
```

Comment:
Can we handle the `null` case inside `getAnomalySeries` - and rename it:
```suggestion
    const anomalyTimeseries = await getAnomalyTimeseries({ 
      ...options, 
      logger, 
      timeseriesDates: latency.avg.map(({ x }) => x) 
    });
```